_posts/2022-07-27-#24858.md

```diff
@@ -10,15 +10,88 @@ status: upcoming
 commit: bcb0cac
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- A new node automatically initializes and populates its data directory by performing
+["Initial block download" (IBD)](https://bitcoin.stackexchange.com/a/106136/97099),
+which is the process of fetching blocks
+from peers and storing them in the `blocks` directory within the data directory
+(default `$HOME/.bitcoin`). 
+
+- The blocks are stored in files named `blknnnnn.dat` (for example, `blk01234.dat`).
+These files are limited to 128 MiB, so each can hold several hundred blocks.
+
+- The `blknnnnn.dat` files are not in any standard database format; their format
+is quite simple: a list of blocks (their serializations), each preceded by
+a "marker" or "magic number" to indicate the start of a block, and the block's
+size or length in bytes. The blocks need not be in height-order, either within a
+block file, or across block files. For example, block 2000 could be stored
+in `blk00010.dat`, while block 1500 could be stored in `blk00011.dat`.
+
+- In order to save disk space, the node operator can enable a mode called
+[_pruning_](https://bitcoin.stackexchange.com/questions/37496/how-can-i-run-bitcoind-in-pruning-mode),
+which causes only the most recent few thousand blocks files to be retained on disk, although
```

Comment:
My pruned node has currently 5 block files on disk, so "a few thousand blocks files" is off. I think the criterion is actually the physical size, i.e. `prune=550`  (550MiB) is the minimum for automatic pruning.

Reply:
Oh thanks, that's a typo, I meant to say "few thousand blocks" (not "blocks files" which isn't even correct grammar). So 500 MiB would be a few hundred blocks, at their current full size. Maybe I'll just say "the most recent few hundred" (I don't want to make it too complicated by mentioning the actual size limit).